Tailviewer.AcceptanceTests/BusinessLogic/LogFiles/TextLogFileAcceptanceTest.cs

```diff
@@ -148,9 +148,9 @@ public void TestDoesNotExist()
 				new Action(() => logFile = new TextLogFile(_scheduler, "dadwdawdw")).ShouldNotThrow();
 
 				logFile.Property(x => x.EndOfSourceReached).ShouldEventually().BeTrue(TimeSpan.FromSeconds(5));
-				logFile.Property(x => x.GetValue(LogFileProperties.EmptyReason)).ShouldEventually().Be(ErrorFlags.SourceDoesNotExist, TimeSpan.FromSeconds(5));
```

Comment:
This merge doesn't look right, GetValue(...) is the new way to retrieve data, please don't revert that.